### PR TITLE
Add titano

### PR DESCRIPTION
**Pull request by tyeth**
Drafting this to check build before review

### CODE REVIEW

**Comment by tyeth** [comment]
As I've reverted the branch of ci-arduino change, I thought there needs to be a merge of https://github.com/adafruit/ci-arduino/pull/177 however the build has completed regardless.

File: src/components/analogIO/Wippersnapper_AnalogIO.cpp [hunk 1]
```diff
@@ -145,6 +145,7 @@ void Wippersnapper_AnalogIO::initAnalogInputPin(
     if (_analog_input_pins[i].enabled == false) {
       _analog_input_pins[i].pinName = pin;
       _analog_input_pins[i].period = periodMs;
+      _analog_input_pins[i].prvPeriod = 0L;
```
Comment (brentru) [inline src/components/analogIO/Wippersnapper_AnalogIO.cpp:148]: Can we do this in a separate PR?

**Review by brentru — CHANGES_REQUESTED** [review]
One question/comment

**Review by brentru — APPROVED** [review]
Approved, LGTM

**Comment by tyeth** [comment]
May not be ready for prime time, or at least I can't get my titano to boot correctly (red light of doom)

**Comment by tyeth** [comment]
This will be included/superseded by #540

**Pull request closed without merging**